s3.py

```diff
@@ -79,6 +78,7 @@ class S3Uploader(MirrorUploader):
     S3_BASE = "https://%s/" % S3_HOSTNAME
 
     BOOK_COVERS_BUCKET_KEY = u'book_covers_bucket'
+    BOOK_BUCKET_KEY = u'book__bucket'
```

Comment:
I've been saying "book bucket" when I actually need the "open_access_content_bucket". We shouldn't need a separate "book bucket" because we'll probably never store non-open-access content on S3.

We do need a 'books' _purpose_ (which indicates that an S3 integration can be used to store open-access books or an LCP integration can be used to store any kind of books), but that's different and you already have that.